pkgs/tools/security/ghidra/build.nix

```diff
@@ -1,33 +1,35 @@
-{ stdenv
-, fetchFromGitHub
-, lib
-, callPackage
-, gradle_7
-, perl
-, makeBinaryWrapper
-, openjdk17
```

Comment:
https://github.com/NationalSecurityAgency/ghidra/blob/5ab72bf4f254773b12c3c9566d14c42e5eb2495f/README.md?plain=1#L49-L50

We should probably also update to JDK 21 and Gradle 8.5+

Reply:
Having just built 11.1.1 for myself, I can confirm it does.